org.mixedrealitytoolkit.input/Interactors/GazePinch/GazePinchInteractor.cs

```diff
@@ -94,7 +94,7 @@ public class GazePinchInteractor :
         /// <summary>
         /// The interactor we're using to query potential gaze pinch targets.
         /// </summary>
-        protected XRBaseInputInteractor DependentInteractor { get => dependentInteractor; set => dependentInteractor = value; }
+        public XRBaseInputInteractor DependentInteractor { get => dependentInteractor; set => dependentInteractor = value; }
```

Comment:
I would like to avoid making this public just for testing purposes.  PL=lease let's find a different solution.

Reply:
You are right, changed to protected internal in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/767/commits/b5240732e09ce6a642878dfb8ef304c92d61a498 , resolving.